### DISCUSSION TITLE
DNT support

### DISCUSSION THREAD

**Discussion by wvholst** [tosdr/edit.tosdr.org#1029] [Ideas]
The W3C DNT standard is a way for browsers to signal a tracking preference. The standard allows for servers to signal back to what extent they will honour such a preference, if at all. The standard also allows for references to ToS-like tracking policies to which a server adheres. The most well-known example of that is the EFF DNT policy document.

So this is something that relates to ToS;DR and it would be nice to see the browser extension test for responses to the DNT:1 signal as well as any policy references.

The W3C DNT repository can be found here: https://github.com/w3c/dnt

**Comment by michielbdejong** [comment]
Good idea! I wouldn't do it from the browser extension, but we could run a script that tests this, and automatically create points accordingly.
